src/community/resources.md

```diff
@@ -17,28 +17,32 @@ See our collection of best and leading practices, common solutions, and more in
 
 To connect with Magento and the Community, join us on the [Magento Community Engineering Slack][]. If you are interested in joining Slack, or a specific channel, send us a request at [engcom@adobe.com](mailto:engcom@adobe.com) or [self signup][].
 
-We have [channels][] for each project, but the following channels are recommended for new members:
+We have [channels][] for each project, Following channels are recommended for new members:
 
--  [announcements][]: Introduce yourself and get quick updates for Magento Community Engineering
--  [general][]: Open chat for introductions and Magento 2 questions
--  [appdesign][]: Open chat for Magento architecture and [technical guidelines][]
--  [coding-standards][]: Open chat for developing [coding standards][] for Magento
--  [github][]: Support for GitHub issues, pull requests, and processes
--  [public-backlog][]: Discussions of the Magento 2 backlog
--  [devdocs][]: Documentation contribution support
+-  [beginners][]: Channel for contribution beginners to ask questions around contribution and share resources.
+-  [announcements][]: Introduce yourself and get quick updates for Magento Community Engineering.
+-  [general][]: Open chat for introductions and Magento 2 questions.
+-  [appdesign][]: Open chat for Magento Architecture and [Technical Guidelines][].
+-  [coding-standards][]: Open chat for developing [Coding Standards][] for Magento.
+-  [github][]: Discussion regarding Pull Request and Issues from the [Magento 2][] repository.
+-  [public-backlog][]: Community discussion around the [Public Backlog][].
+-  [devdocs][]: Discussions of the documentation impact of Community Contributions.
 
 [contribute]: https://github.com/magento/devdocs/blob/master/.github/CONTRIBUTING.md
-[technical guidelines]: {{ site.baseurl }}/guides/v2.3/coding-standards/technical-guidelines.html
-[coding standards]: {{ site.baseurl }}/guides/v2.3/coding-standards/bk-coding-standards.html
+[Public Backlog]: https://github.com/magento/backlog
```

Comment:
The links you have added are Title Case rather than lowercase (the ones previously here)
Since you have done this change, I wonder if it makes sense to take the opportunity to change them all so they are all Title Case 🤔

Reply:
Thank you @diazwatson , Should I make them all in Title Case or all in lowercase?

Reply:
waiting for your response @diazwatson

Reply:
These are mostly Slack channel names, which tend not to be capitalized. The capitalized links are page titles, which is correct.